internal/ethapi/api.go

```diff
@@ -806,7 +806,7 @@ func DoCall(ctx context.Context, b Backend, args CallArgs, blockNrOrHash rpc.Blo
 		}
 	}
 	// Set default gas & gas price if none were set
-	gas := uint64(math.MaxUint64 / 2)
+	gas := uint64(10000000)
```

Comment:
Will break with max gas because EM asserts that gas is < `maxGasPerQueuePerEpoch` (not sure what the right value is to put here -- block gas limit?)

Reply:
I'm using `ExecutionManager.GetMaxTransactionGasLimit`